.github/pull_request_template.md

```diff
@@ -12,6 +12,7 @@ Fixes #    <!-- Add the issue number that is fixed by this PR -->
 <!-- Use '[x]' to indicate checked box in the list below -->
 ## Checklist:
 
+- [ ] Read the contributing guidelines
```

Comment:
Can you add a link to the guidelines here, I'm guessing that can only be done post merging this PR. Let's keep that in mind.